freqtrade/configuration.py

```diff
@@ -36,6 +36,9 @@ def load_config(self) -> Dict[str, Any]:
         # Add the strategy file to use
         config.update({'strategy': self.args.strategy})
 
+        if self.args.strategy_path:
+            config.update({'strategy_path': self.args.strategy_path})
```

Comment:
Does this mean the path can also be configured in the settings (which would be great!)? 
if so, you should also add it in `config_full.json.example` for reference ...

Reply:
Good catch! :-) Yes, I think it should work, I will write a test for that and add it to the full example